lib/src/components/NxToggle/NxToggle.scss

```diff
@@ -66,6 +66,17 @@
         margin-left: 20px;
       }
     }
+
+    &:disabled {
```

Comment:
This block duplicates styles that are present above.  I think rather than repeating them you just need to increase the specificity of those styles

Reply:
> This block duplicates styles that are present above. I think rather than repeating them you just need to increase the specificity of those styles

Removed repeated styles and increased specificity.
[f11153f](https://github.com/sonatype/sonatype-react-shared-components/commit/f11153fe37e86b92b2a0bd918c3d4acc789f689a)